ocaml/xapi/helpers.ml

```diff
@@ -39,15 +39,21 @@ let log_exn_continue msg f x =
   with e ->
     debug "Ignoring exception: %s while %s" (ExnHelper.string_of_exn e) msg
 
-let call_script ?(log_successful_output = true) ?env script args =
+let call_script ?(log_successful_output = true)
```

Comment:
Is this calling for a type to capture this and the default `SuccessOnly`? I'd call the parameter `logging` that would control this.

```
type log = Always | Never | SuccessOnly | ErrorOnly
```

Reply:
I thought there would be a lot more usages of `call_script`, but it wasn't that bad to change the interface